src/app/exam/problems/ExamProblems.js

```diff
@@ -0,0 +1,171 @@
+import {useState, useEffect, useCallback} from "react";
+import { useParams, useRouteMatch } from "react-router-dom";
+
+import { examService, submissionService } from "../../../services/services.js";
+import { ThreeDotsButton } from "../../commons/buttons/ThreeDotsButton.js";
+import FakeLink from "../../commons/FakeLink.js";
+import { ItemListPage } from "../../commons/ItemListPage/ItemListPage.js";
+import { ExamInPageNavigationBar } from "../ExamInPageNavigationBar";
+import { AddProblemModal } from "../modals/AddProblemModal.js";
+import { EditProblemModal } from "../modals/EditProblemModal.js";
+import {RejudgeProblemModal} from "../modals/RejudgeProblemModal";
+import "./ExamProblems.scss";
+
+const toCharacterIndex = i => {
+    return String.fromCharCode(i + 65);
+}
+
+
+const ExamProblems = ({ exams }) => {
+    const { url: currentURL } = useRouteMatch()
+    const { examId } = useParams()
+    const currentExamName = exams.find(exam => exam.id === parseInt(examId))?.name
+    const [examProblems, setExamProblems] = useState(null);
```

Comment:
Don't understand, why all exams are passed in.

Just pass the current exam in.